aiidalab_widgets_base/static/styles/README.md

```diff
@@ -0,0 +1,15 @@
+# Stylesheets for AiiDAlab Widgets Base
+
+This folder contains the following:
+
+- `scss` - a folder for SCSS-format stylesheets to be compiled
+- `.css` files  - compiled CSS stylesheets
+
+It is recommended to add SCSS stylesheets in the SCSS folder for functionality and flexibility (see https://sass-lang.com/).
+
+Compiling SCSS into CSS can be done using:
```

Comment:
Does this mean that compiled CSS files should be checked in the repo? If so, we need a pre-commit hook that will check that the files are up-to-date. Alternatively, we need to create a hook that is called when the package is installed.

Reply:
I gotta be honest, I'm not all that fond of the way I did it. But I couldn't come up with an alternative. It would be great to implement something more flexible, where one can use both CSS and SCSS. Let's chat on Discourse.